CMakeLists.txt

```diff
@@ -69,6 +69,13 @@ endif()
 set(ENABLE_UPDATE_CHECKER (NOT ${DEVELOPMENT_BUILD}) CACHE BOOL
 	"Whether to enable update checks by default")
 
+find_program(CCACHE_FOUND ccache)
+if (CCACHE_FOUND)
+    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
+    set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ccache)
```

Comment:
I've found several sources that one should not set this to ccache, as ccache does not cache linking:
http://github.com/IntelRealSense/librealsense/issues/9393
https://crascit.com/2016/04/09/using-ccache-with-cmake/
```suggestion
```

Reply:
fix it